.gitignore

```diff
@@ -134,4 +134,8 @@ gradle-app.setting
 ### Gradle Patch ###
 **/build/
 
-# End of https://www.gitignore.io/api/java,gradle,intellij
\ No newline at end of file
+# End of https://www.gitignore.io/api/java,gradle,intellij
+
+.idea/vcs.xml
+.idea/jarRepositories.xml
+.idea/compiler.xml
```

Comment:
I suggest we keep IDE config files out of the repo (even though this is an IDE project :D)

Reply:
oops!

Reply:
No, I mean, there were no such files in our repo. My IDE just generated some files that would had been part of this commit but I decided to put those in gitignore instead. Apparently it is arguable if files such as `.idea/misc.xml` should be in VCS or not but I think it's better not have them.

Reply:
ahhh... now I understand :) (I first thought I've pushed some "tech" files myself)